opentech/apply/activity/templates/messages/email/contract_uploaded.html

```diff
@@ -0,0 +1,9 @@
+{% extends "messages/email/base.html" %}
+{% block salutation %}Dear Applicant,{% endblock %}
```

Comment:
There is an `applicant_base.html` that would probably be worth using here?

Reply:
Fixed and rebased.